src/main/java/model/datafeed/DatafeedV2.java

```diff
@@ -0,0 +1,26 @@
+package model.datafeed;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties()
```

Comment:
is the annotation used if there are no parameters?

Reply:
yes